src/components/Comment/ButtonInteraction.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+import './ButtonInteraction.css';
+
+class ButtonInteraction extends Component {
+    constructor() {
+        super();
+        this.state = {
+            counter: 0
```

Comment:
I hope it makes sense why we wouldn't want to keep state here. We want to "lift state up" and keep our single source of truth in a higher component.